src/file-upload/file-list.tsx

```diff
@@ -178,7 +180,13 @@ export const FileList = ({
     };
 
     const shouldRenderEditMode = (item: FileItemProps) => {
-        return checkEditable(item) && !item.description;
+        return (
+            !item.errorMessage &&
+            !readOnly &&
+            !(item.progress && item.progress < 1) &&
```

Comment:
handle other falsy values
```suggestion
            !(item.progress < 1) &&
```

Reply:
This will allow value like `null` to be assessed. Since we are only expecting numbers, will change to something more robust like a type check